test/e2e/serial/tests/scheduler_cache.go

```diff
@@ -612,12 +597,25 @@ var _ = Describe("[serial][scheduler][cache][tier1] scheduler cache", Label("sch
 				// NRT updater, resync loop, scheduler retry loop.
 				failedPods, updatedPods = wait.With(fxt.Client).Timeout(5*time.Minute).ForPodListAllRunning(context.TODO(), expectedRunningPods)
 				Expect(len(updatedPods)).To(Equal(hostsRequired))
+				dumpFailedPodInfo(fxt, failedPods)
```

Comment:
In case of failed assertion in the line above, we won't get this information.

Reply:
good catch, let me address this